src/clients/api/queries/getVaiTreasuryPercentage.ts

```diff
@@ -0,0 +1,21 @@
+import BigNumber from 'bignumber.js';
+
+export interface IGetVaiTreasuryPercentageInput {
+  vaiControllerContract: $TSFixMe; // @TODO: use contract type (through Typechain?)
+}
+
+export type IGetVaiTreasuryPercentageOutput = number;
```

Comment:
This is not an interface so it should not be prefaced with an `I`

Reply:
You're right. That applies to the types of other queries and mutations as well. I'll make the change 👍